examples/datasets/plot_limo_data.py

```diff
@@ -75,18 +74,17 @@
 #
 # In the original LIMO experiment (see [2]_), participants performed a
 # two-alternative forced choice task, discriminating between two face stimuli.
-# Subjects discriminated the same two faces during the whole experiment.
-# However, the critical manipulation consisted in the level of noise
-# added to the face-stimuli during the task, making the faces more or less
+# The same two faces were used during the whole experiment,
+# with varying levels of noise added, making the faces more or less
 # discernible to the observer (see `Fig 1`_ in [3]_ for instance).
 #
 # The presented faces varied across a noise-signal (or phase-coherence)
 # continuum spanning from 0 to 100% in increasing steps of 10%.
 # In other words, faces with high phase-coherence (e.g., 90%) were easy to
 # identify, while faces with low phase-coherence (e.g., 10%) were hard to
 # identify and by extension very hard to discriminate.
-
-###############################################################################
+#
+#
```

Comment:
you only need the full row of 79 `#`s when you're transitioning from a code block to a rST block.

Reply:
Alrigth, thanks. I'll pay more attention to these in the future.